engine/apps/google/client.py

```diff
@@ -2,16 +2,28 @@
 import logging
 import typing
 
+import pytz
 from django.conf import settings
 from google.oauth2.credentials import Credentials
 from googleapiclient.discovery import build
 
-from apps.google.types import GoogleCalendarEvent
+from apps.google import constants, utils
+from apps.google.types import GoogleCalendarEvent as GoogleCalendarEventType
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
 
+class GoogleCalendarEvent:
+    def __init__(self, event: GoogleCalendarEventType):
+        self.raw_event = event
+        self._start_time = utils.datetime_strptime(event["start"]["dateTime"])
+        self._end_time = utils.datetime_strptime(event["end"]["dateTime"])
+
+        self.start_time_utc = self._start_time.astimezone(pytz.UTC)
```

Comment:
FYI, if possible we should try to move to Python [zoneinfo](https://docs.python.org/3/library/zoneinfo.html#module-zoneinfo) while limiting/deprecating the use of [pytz](https://github.com/stub42/pytz/blob/master/src/README.rst#issues--limitations) (which django is also deprecating since [4.0](https://docs.djangoproject.com/en/5.0/releases/4.0/#zoneinfo-default-timezone-implementation))

Reply:
good call, thanks for pointing that out 👍